test/Microsoft.AspNetCore.Razor.Test.MvcShim/Microsoft.AspNetCore.Razor.Test.MvcShim.csproj

```diff
@@ -3,13 +3,13 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net46;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
```

Comment:
Why the change here? I'd prefer to keep these in sync with the src projects, but it's not a huge deal if there's a good reason why we need to lift it.